### PR TITLE
Edited readme

### PR DESCRIPTION
**Pull request by NakulSabharwal**
- Added installation guide using gradle
- Added sample

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,283 @@
...
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="your.domain.name">
+
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
```
Comment (deepak2016) [inline README.md]: why are you putting everything from manifest file in the readme, only put relevant part in the readme, remove the common part.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,283 @@
...
+import android.os.AsyncTask;
+import android.os.Build;
+import android.os.StrictMode;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
```
Comment (deepak2016) [inline README.md]: Again here in code, remove common part from readme, only put the part which is relevant and would be useful for someone using the library.

**Review by deepak2016 — COMMENTED** [review]
Remove portion of code in readme which is common for applications. Only keep important/relevant and code which would have to change.

File: README.md (hunk 1 above)
Comment (NakulSabharwal) [reply README.md]: removed sample from readme.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,155 @@
...
+
+## 6. Contributions
+
+The Microsoft Graph SDK is open for contribution. To contribute to this project, see [Contributing](https://github.com/microsoftgraph/msgraph-sdk-java/blob/master/CONTRIBUTING.md).
+
+Thanks to everyone who has already devoted time to improving the library:
+
+<!-- ALL-CONTRIBUTORS-LIST:START  -->
+<!-- prettier-ignore -->
+| [<img src="https://avatars3.githubusercontent.com/u/16473684?v=4" width="100px;"/><br /><sub><b>Nakul Sabharwal</b></sub>](https://developer.microsoft.com/graph)<br />[](#question-NakulSabharwal "Answering Questions") [](https://github.com/microsoftgraph/msgraph-sdk-java/commits?author=NakulSabharwal "Code") [](https://github.com/microsoftgraph/msgraph-sdk-java/wiki "Documentation") [](#review-NakulSabharwal "Reviewed Pull Requests") [](https://github.com/microsoftgraph/msgraph-sdk-java/commits?author=NakulSabharwal "Tests")| [<img src="https://avatars2.githubusercontent.com/u/3197588?v=4" width="100px;"/><br /><sub><b>Deepak Agrawal</b></sub>](https://github.com/deepak2016)<br />
```
Comment (deepak2016) [inline README.md]: point to msgraph-sdk-android-auth repository

File: README.md [hunk 4]
```diff
@@ -1,2 +1,155 @@
...
+     ...
+     // Handle failure case
+  });
+```
+
+## 4. Documentation
+
+For more detailed documentation, see:
+
+* [Overview](https://github.com/microsoftgraph/msgraph-sdk-java/wiki/Overview)
```
Comment (deepak2016) [inline README.md]: use msgraph-sdk-android-auth wiki, if not applicable remove msgraph-sdk-java links

File: README.md (hunk 2 above)
Comment (NakulSabharwal) [reply README.md]: I removed this code, will move it in wiki in next PR.

File: README.md (hunk 3 above)
Comment (NakulSabharwal) [reply README.md]: changed link to android auth

File: README.md (hunk 4 above)
Comment (NakulSabharwal) [reply README.md]: removed it currently, will add it in next pr

**Pull request closed without merging**